html/pfappserver/root/static.alt/src/views/Configuration/_router/index.js

```diff
@@ -6,6 +6,7 @@ import DomainsStore from '../_store/domains'
 import RealmsStore from '../_store/realms'
 import AuthenticationSourcesStore from '../_store/sources'
 import PortalModulesStore from '../_store/portalmodules'
+import BillingTiersStore from '../_store/billing_tiers'
```

Comment:
We're currently omitting underscores in store names. To be consistent would should either continue to omit them, or switch them to all to camel-case.

Reply:
IMO we should use either underscores or camel casing.

Personally, I don't think names like the one below are readable:
/configuration/policesaccesscontrol

So, should we switch, should I go ahead and do it now in this PR, or is this something that is better addressed through yours since you're reworking a bunch of stuff at the same time ?